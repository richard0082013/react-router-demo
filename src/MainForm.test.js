import React from "react";
import renderer from "react-test-renderer";
import MainForm from "./components/mainForm";

it("renders correctly when load", () => {
  const tree = renderer.create(<MainForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
