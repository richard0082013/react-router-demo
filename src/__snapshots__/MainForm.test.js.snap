// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when load 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  role="button"
>
  Let's get it started! 
</button>
`;
